json-endpoints/class.wpcom-json-api-list-posts-v1-1-endpoint.php

```diff
@@ -311,6 +312,13 @@ function callback( $path = '', $blog_id = 0 ) {
 						$return[$key]->next_page = $this->build_page_handle( $last_post, $query );
 					}
 				}
+
+				if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+					if ( !isset( $return[$key] ) )
```

Comment:
Should we take that opportunity to clean up that code so it respects the WP coding standards?